packages/larva-patterns/algorithms/a-carousel-grid/a-carousel-grid.overlay.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
```

Comment:
Shouldn't this be in the prototype since it's the only variation?

Reply:
The `algorithms` patterns are purely for prototyp_ing_ (not `prototype`) purposes - there isn't data associated with them, only the HTML required express what classes and structural markup are needed. The `a-carousel-grid` has a variation of `overlay`,  so this file represents what classes are layered onto to the original based `lrv-a-carousel-grid` selectors to get the variation. For other CSS algorithms this is usually a single class, but the variations here re-use few of the same declarations, so it requires more additional classes. This is something @tinuola and I talked about revisiting later to see if there's a more concise solution in the CSS.